bin/punic-build-data

```diff
@@ -72,6 +74,9 @@ try {
         $sourceData->ensureJsonLocale('en');
         echo "done.\n";
     }
+    if (!class_exists('\Collator')) {
```

Comment:
@c960657 Is this required? Punic `Comparer` class should handle the case when it's missing, isn't it?

Reply:
`Punic\Comparer` uses an approximated comparison method that is of lower quality than that in `Collator`.

Most users wont need to run this script, and they don't need to run in on their production server, so I think the `intl` extension is a modest requirement.

Reply:
I do agree